src/nunit.analyzers/TestMethodUsage/TestMethodUsageAnalyzer.cs

```diff
@@ -78,19 +78,24 @@ public override void Initialize(AnalysisContext context)
         {
             context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
             context.EnableConcurrentExecution();
-            context.RegisterSymbolAction(AnalyzeMethod, SymbolKind.Method);
+            context.RegisterCompilationStartAction(AnalyzeCompilation);
         }
 
-        private static void AnalyzeMethod(SymbolAnalysisContext context)
+        private static void AnalyzeCompilation(CompilationStartAnalysisContext context)
```

Comment:
Please rename to `AnalyzeCompilationStart`

Reply:
```suggestion
        private static void AnalyzeCompilationStart(CompilationStartAnalysisContext context)
```